uisdk/src/main/java/com/karhoo/uisdk/screen/booking/booking/payment/adyen/AdyenDropInServicePresenter.kt

```diff
@@ -89,6 +89,10 @@ class AdyenDropInServicePresenter(service: AdyenDropInServiceMVP.Service,
         payload.put(RETURN_URL, returnUrl)
         payload.put(CHANNEL, ANDROID)
 
+        val additionalData = JSONObject()
+        additionalData.put(ALLOW_3DS, true)
```

Comment:
Would it be better to make it a Constant value? so it would be more readable and easy to parametrise it later (might never happen)

e.g. additionalData.put(ALLOW_3DS, ALLOW_3DS_TRUE)

Reply:
Done